### ISSUE TITLE
Unable to find cookbook for file on Chef 12.9.38

### ISSUE THREAD

**Issue by danmelton** [poise/poise-ruby-build#3]
Hi,

In the latest version of Chef (12.9.38) on ubuntu 14.04 x86_64, I'm receiving this error:

```
Unable to find cookbook for file "/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.9.38/lib/chef/mixin/lazy_module_include.rb"
```

from this codeblock:

```
ruby_runtime 'somename' do
  provider :ruby_build
  version '2.2'
end
```

My metadata file:

```
depends 'poise-ruby'
depends 'poise-ruby-build'
```

When I rollback chef to 12.6.0 (previously working state of the same code), it installs no problem.

Full MetaData Dependencies:

depends 'nginx'
depends 'monit'
depends 'poise-ruby'
depends 'poise-ruby-build'
depends 'application_ruby', '=3.0.2'
depends 'application', '=4.1.6'
depends 'application_procfile', '=0.3.38'
depends 'remote_syslog2'
depends 'chef_handler'
depends 'slack_handler'

```
=> default:     /tmp/vagrant-chef/085aa9d20a9911d1d4713389f3f11d6c/cookbooks/poise-ruby/files/halite_gem/poise_ruby/ruby_providers/base.rb:43:in `action_install'
==> default:
==> default:     Resource Declaration:
==> default:     ---------------------
==> default:     # In /tmp/vagrant-chef/085aa9d20a9911d1d4713389f3f11d6c/cookbooks/neighborly/recipes/default.rb
==> default:
==> default:      12: ruby_runtime 'somename' do
==> default:      13:   provider :ruby_build
==> default:      14:   version '2.2'
==> default:      15: end
==> default:      16:
==> default:
==> default:     Compiled Resource:
==> default:     ------------------
==> default:     # Declared in /tmp/vagrant-chef/085aa9d20a9911d1d4713389f3f11d6c/cookbooks/neighborly/recipes/default.rb:12:in `from_file'
==> default:
==> default:     ruby_runtime("somename") do
==> default:       provider PoiseRuby::RubyBuild::Provider
==> default:       action [:install]
==> default:       retries 0
==> default:       retry_delay 2
==> default:       default_guard_interpreter :default
==> default:       subresources [ruby_gem[bundler]]
==> default:       declared_type :ruby_runtime
==> default:       cookbook_name :neighborly
==> default:       recipe_name "default"
==> default:       version "2.2"
==> default:     end
==> default:
==> default:     Platform:
==> default:     ---------
==> default:     x86_64-linux
```

**Comment by coderanger** [comment]
This was fixed in Poise 2.6.1.

**Comment by danmelton** [comment]
@coderanger Thanks, I'll double check my cookbook versions and reopen if the issue exists.
